torch/onnx/_internal/exporter/_dynamic_shapes.py

```diff
@@ -42,7 +42,8 @@ def from_dynamic_axes_to_dynamic_shapes(
 
     warnings.warn(
         "from_dynamic_axes_to_dynamic_shapes is deprecated and will be removed in a future release. "
-        "Please provide 'dynamic_shapes' directly.",
+        "This function converts 'dynamic_axes' format (including custom axis names) to 'dynamic_shapes' format. "
+        "Instead of relying on this conversion, provide 'dynamic_shapes' directly using torch.export.Dim objects with custom names.",
```

Comment:
We do support strings right? Maybe just say supply dynamic_shapes?

Reply:
Removed "torch.export.Dim objects"